tests/xharness/Jenkins.cs

```diff
@@ -892,6 +893,11 @@ public int Run ()
 			get { return Harness.JenkinsConfiguration == "server"; }
 		}
 
+		void BuildTestLibraries ()
+		{
+			ProcessHelper.ExecuteCommandAsync ("make", $"all -j{Environment.ProcessorCount} -C {StringUtils.Quote (Path.Combine (Harness.RootDirectory, "test-libraries"))}", MainLog, TimeSpan.FromMinutes (1)).Wait ();
+		}
```

Comment:
We should return a Task, not void.

Reply:
I call `.Wait ()` at the end to make it synchronous.